qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -155,6 +158,15 @@ def __init__(
             self._api_client = RuntimeClient(self._client_params)
             self._backend_allowed_list = self._discover_cloud_backends()
         else:
+            warnings.warn(
+                'The "ibm_quantum" channel option is deprecated and will be sunset on 1 July. '
+                "After this date, ibm_cloud will be the only valid channel. "
```

Comment:
@kt474, isn't `local` also a valid channel after July 1?

Reply:
Yeah that was an oversight - I'll update this message and backport